djangochannelsrestframework/observer/base_observer.py

```diff
@@ -33,6 +35,26 @@ def serialize(self, signal, *args, **kwargs) -> Dict[str, Any]:
         return message
 
     def serializer(self, func):
+        """Adds a Serializer to the model observer return.
+
```

Comment:
Thankyou so much for adding these docs. 

If you want to add some more detail here: 

might be worth adding something about how this serializer is the serializer used to send the object over the channel layer. 

This means it is called once per update but also means it `does not have access to the subscribed user` so you can't do any filtering based on that. If you need to do user centric serialisation you should do that in: 

```python
@model_observer(Comments)
 async def comment_activity(self, message, observer=None, **kwargs):
     ## do user specific filtering etc here.
     await self.send_json(message)
```

--

however also feel free to ignore this I can come back later and fill these things in now that we have something in place it will be much simpler to add these details.

Reply:
Ok! no problem! I will add it

Reply:
This is something that many people seem to get confuses about asking for access to the User, I should have maybe used a different name like `encoder` or something so that it was not using the same terminology.

Reply:
Check the new example, you lost me with the encoder xD